pkg/customflag/validate.go

```diff
@@ -140,8 +140,8 @@ func validateMultusTest(expectedValue, valuesUpgrade []string) {
 }
 
 func validateComponentsTest(expectedValue, valuesUpgrade []string) {
-	k3scomponentsCmdsLength := 9
-	rke2componentsCmdsLength := 8
+	k3scomponentsCmdsLength := 10
```

Comment:
is there a way to make it dynamic instead of static? people may forget to update this as new components gets added...

Reply:
not seeing right now a good and nice way of doing that , rather than create another env var `CMD_LENGHT` which i was avoiding, and this is not so regularly updated, or it shouldnt.

the ideal would be now on this review we bring more components we may want and already add it now.